src/actions/party.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/prefer-default-export */
+
+import types from './types';
+import Db from '../db';
+import NavigationService from '../navigation';
+
+// TODO: Subscribe/Unsubscribe to party
+const noop = () => {};
+
+let unsubscribeParty = noop;
+
+export const setParty = party => ({
+  type: types.party.set,
+  party,
+});
+
+export const unsubscribe = () => {
```

Comment:
You should assign `noop` to `unsubscribeParty` after unsubscribing.

Reply:
Good one, it's done in line 18.